service_monitoring_validation/service_monitoring_sample/myservice-slo-availability.json

```diff
@@ -0,0 +1,21 @@
+{
+  "criteria": [
+    {
+      "target": 95,
+      "timeframeFrom": "now-7d",
+      "timeframeTo": "now"
+    }
+  ],
+  "description": "Measures the proportion of successful service requests over time.",
+  "name": "{{ .name }}",
+  "sliReference": {
+    "templateId": "e2J1aWx0aW46aW50ZXJuYWwuc2VydmljZS5sZXZlbC5vYmplY3RpdmUudGVtcGxhdGVzLCBpZDogYTU2NWMzNGEtYzVhNS00MTE1LTkzMTEtMDcxYjRjYjU5ZGQ3fQ==",
```

Comment:
@g3kl  should we remove the reference to a `templateId` and rather use a custom SQL?